bcgov/foi-flow
# Changes to handle assignment based on ministry user type and on closu…

Change Details:
1. Changes to handle role based notifications for assignment event
2. On closure, delete all notifications associated with request.

Test Cases:
1. Create request and assign to Flex team member in CFR state. Ensure ministry login do not have notification. 
2.  Login as ministry user and assign the request to team member. This should not create notification for Flex team member
3. Closing the request should automatically dismiss all older assignment and watching notifications of state change for users.

Sanity Checks Performed:
1. Create new request
2.  Update Request as IAO, Flex and ministry
3.  Handle state navigation Unopened -> Open -> CFR -> Closed -> CFR
4. Comments add & reply.
